import-chrome/src/import-chrome.cpp

```diff
@@ -158,14 +177,23 @@ int main( int argc, char** argv )
         {
             for ( auto& kv : v["args"].items() )
             {
-                zoneText += kv.key() + ": " + kv.value().dump() + "\n";
+                const auto val = kv.value();
+                const std::string s =
+                    val.is_string() ?
+                    val.get<std::string>() : val.dump();
+                zoneText += kv.key() + ": " + s + "\n";
```

Comment:
In case you have a rich string in the argument (say with newlines inside), this decodes the json string to turn "\\n" into a regular '\n' that will be rendered on multiple lines in Tracy.